CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Version 0.4.0-dev (changes since 0.4.0 release go here)
 
 ### Features
+* Added ``skbio.io.format.blast6.py`` module as a new format that reads blast6 data to a pandas dataframe. ([#1110](https://github.com/biocore/scikit-bio/issues/1110))
```

Comment:
Can you change this to read:

```
Added `skbio.io.format.blast6` for reading BLAST+ output format 6 or BLAST output format 8 files into a `pd.DataFrame`.
```

I basically want the wording to be as precise as possible since there is often confusion about different BLAST output formats.